src/dotnet/commands/dotnet-run3/Run3Command.cs

```diff
@@ -80,21 +84,16 @@ private ICommand GetRunCommand()
                 globalProperties.Add("TargetFramework", Framework);
             }
 
-            ProjectInstance projectInstance = new ProjectInstance(Project, globalProperties, null);
+            Project project = new Project(Project, globalProperties, null);
 
-            string runProgram = projectInstance.GetPropertyValue("RunCommand");
+            string runProgram = project.GetPropertyValue("RunCommand");
```

Comment:
As long as it says: since it isn't built. That's fine by me. We just need to tell the user that a build is needed.